Formula/mdcat.rb

```diff
@@ -22,8 +25,20 @@ class Mdcat < Formula
     depends_on "pkg-config" => :build
   end
 
+  resource "asciidoc" do
+    url "https://files.pythonhosted.org/packages/8a/57/50180e0430fdb552539da9b5f96f1da6f09c4bfa951b39a6e1b4fbe37d75/asciidoc-10.2.0.tar.gz"
+    sha256 "91ff1dd4c85af7b235d03e0860f0c4e79dd1ff580fb610668a39b5c77b4ccace"
+  end
```

Comment:
We should do `depends_on "asciidoc"` instead.

Reply:
`depends_on` will bring in a plethora of dependencies, which I would like to avoid.

Reply:
The only way to avoid that would be not to install the manpage then. Adding a formula-specific copy of `asciidoc` is something we need to avoid. See https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae.

Reply:
> `depends_on` will bring in a plethora of dependencies, which I would like to avoid.

I think `depends_on "asciidoc" => :build` should be enough. Then it won't bring any dependencies

Reply:
> The only way to avoid that would be not to install the manpage then. Adding a formula-specific copy of `asciidoc` is something we need to avoid. See https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae.

Thanks for providing more information on this case. If an [exception](https://docs.brew.sh/Acceptable-Formulae#sometimes-there-are-exceptions) can't be made in this case, I don't mind closing this PR. Cheers.


> > `depends_on` will bring in a plethora of dependencies, which I would like to avoid.
> 
> I think `depends_on "asciidoc" => :build` should be enough. Then it won't bring any dependencies

`:build` will bring in the dependencies, if I ever have to build the package locally.

Reply:
Avoiding build dependencies doesn't warrant an exception.

Thank you for the PR anyway.